models/noise_generator.h

```diff
@@ -76,38 +76,42 @@ If the modulation is added the current is given by
                               \text{ for } t_0 + j dt \leq t < t_0 + (j-1) dt
 
 For a detailed discussion of the properties of the noise generator, please see
-the ``noise_generator.ipynb`` notebook included in the NEST source code
-(``docs/model_details``).
+`noise_generator <../model_details/noise_generator.ipynb>`_
+notebook included in the NEST source code.
 
 Remarks:
-- All targets receive different currents.
-- The currents for all targets change at the same points in time.
-- The interval between changes, dt, must be a multiple of the time step.
-- The effect of this noise current on a neuron DEPENDS ON DT. Consider
-  the membrane potential fluctuations evoked when a noise current is
-  injected into a neuron. The standard deviation of these fluctuations
-  across an ensemble will increase with dt for a given value of std.
-  For the leaky integrate-and-fire neuron with time constant :math:`\tau_m` and
-  capacity :math:`C_m`, membrane potential fluctuations Sigma at time
-  :math:`t_j+delay` are given by
+
+ - All targets receive different currents.
+
+ - The currents for all targets change at the same points in time.
+
+ - The interval between changes, dt, must be a multiple of the time step.
+
+ - The effect of this noise current on a neuron depends on DT. Consider
+   the membrane potential fluctuations evoked when a noise current is
+   injected into a neuron. The standard deviation of these fluctuations
+   across an ensemble will increase with dt for a given value of std.
+   For the leaky integrate-and-fire neuron with time constant :math:`\tau_m` and
+   capacity :math:`C_m`, membrane potential fluctuations Sigma at time
+   :math:`t_j+delay` are given by
 
 .. math::
```

Comment:
Can this and the next ``.. math::`` block be indented, so that they become part of the list bullet point?